tests/e2e/conf/stage.conf

```diff
@@ -66,3 +66,10 @@ TEST_SUITE_RUN=basic
 # Testcase variables
 PLUGIN_NAME=ibm.io/ibmc-block
 PVC_ACCESS_MODE=RWO
+
+# Volume creation varibles
+OLD_REQUEST_URL=containers.bluemix.net
+NEW_REQUEST_URL=containers.stage1.bluemix.net
+OLD_REGION="region: us-east"
+NEW_REGION="region: $ARMADA_REGION"
```

Comment:
You dont need the `$ARMADA_REGION` variable. Instead you can set the region as `us-south` here as this conf file is specific to stage which is in us-south region.